indico/htdocs/sass/partials/_core.scss

```diff
@@ -104,3 +104,15 @@
     overflow-x: hidden;
     overflow-y: auto;
 }
+
+.transparent-overlay {
+    position: relative;
+    &:before {
+        @include gradient(rgba(0, 0, 0, 0.0), white);
```

Comment:
Empty line after this line
https://github.com/indico/indico/wiki/SASS-Guidelines